bodhi/consumers/masher.py

```diff
@@ -378,6 +379,9 @@ def work(self):
             self.check_all_karma_thresholds()
             self.obsolete_older_updates()
 
+            # Update datetime for locking on update
+            self.locked_date_for_update()
```

Comment:
So this will set the `locked_date` for each update *after* the push is done. Don't we want to set this when it is initially locked, at the beginning of the `work` method?